sdk/core/azure-core/src/credentials/credentials.cpp

```diff
@@ -0,0 +1,12 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+#include "azure/core/credentials/credentials.hpp"
+
+using Azure::Core::Credentials::TokenCredential;
+
+namespace {
```

Comment:
I'll reiterate my comment about anonymous namespaces. IMHO these are better handled as static members of the class (functionally the two are identical, but making them as static members makes their usage as a member of the class explicit).